contents/docs/features/actions.md

```diff
@@ -16,101 +16,99 @@ For example, a typical action might be one of the following:
 
 ![](../../images/03/Posthog-17.png)
+<br>
 
-## Creating front end Actions
+## Creating Frontend Actions
+<br>
 
 ### Pre-Requisites
 
-Before creating an Action, you must have provided the domain you will be using PostHog on. This should be the root domain where your application is hosted, ie http://example.com. Do this by visiting Setup > Setup your PostHog account.
+Before creating an Action, you must have provided the domain you will be using PostHog on. This should be the root domain where your application is hosted e.g. http://example.com. Do this by visiting 'Setup' > 'Setup your PostHog account'.
+<br>
 
-### Getting started
+### Getting Started
 
-The simplest way to create a front-end based Action is to visit your application, and to use PostHog’s toolbar to tag the elements that are relevant.
+The simplest way to create a frontend-based Action is to visit your application, and to use PostHog’s toolbar to tag the elements that are relevant.
 
-To create a new Action, click 'New Action' button at the top right of the Actions page:
+To create a new Action, click the blue 'New Action' button on the above the table on the Actions page:
 
-![](../../images/03/Posthog-18.png)
+![](../../images/actions-page.png)
 
-This will open a the New Action page with the 3 types of Actions you can record:
+This will open the following page:
 
-* Frontend Element
-* Custom Event
-* Page view
+![](../../images/create-action.png)
+<br>
 
-![](../../images/03/Posthog-19.png)
+Here, you have a choice of two options:
 
-Page views can match urls containing a string or match exactly.
+#### Inspect element on your site
 
-![](../../images/03/Posthog-20.png)
+This will open the domain you provided to PostHog so that you can select an element with our Inspect tool and create an action from this.
 
-Custom Events can be sent to your PostHog instance by API we have several libraries to allow you to do [this](/Integrations).
+> **Note:** Our [Toolbar](/docs/features/toolbar) is making this significantly easier.
 
-Frontend Elements can be selected directly on your site, click 'Select element on site'.
+#### From event or pageview
 
-![toolbar pop up](../../images/03/Posthog-21.png)
+Alternatively you can create an event based on `pageview` or an existing [event](/docs/features/events) on your website.
 
-This will pop open the domain you provided in Setup.
+Choosing this option will open the following page where you can create your desired Action:
 
-You will see a PostHog toolbar pop up on the right hand side:
+![](../../images/action-from-event.png)
+<br>
 
-![PostHog Toolbar](../../images/02/Screenshot-2020-02-09-at-14.48.17.png)
+ 
+#### Note: Actions Containing Multiple Events 
 
+It is possible for an action to match multiple events. To do this, click ‘Add another match group’ when you are creating your action (see picture above).
```

Comment:
This could be expanded on - I can't remember if it's OR, or AND logic here. Could you check + clarify please?

Reply:
I believe I read somewhere else that this was AND. That would also make more sense i think. Will double-check

Reply:
Actually they are OR. Mentioned on the app itself